internal/pgaudit/postgres.go

```diff
@@ -0,0 +1,73 @@
+/*
+ Copyright 2021 Crunchy Data Solutions, Inc.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package pgaudit
+
+import (
+	"context"
+	"strings"
+
+	"github.com/crunchydata/postgres-operator/internal/logging"
+	"github.com/crunchydata/postgres-operator/internal/postgres"
+)
+
+// When the pgAudit shared library is not loaded, the extension cannot be
```

Comment:
:+1: this explanation

Reply:
I wasn't sure where to put it. It seemed too small for its own markdown file.